src/entryPoints/AuthModule/AuthUtils.res

```diff
@@ -25,3 +25,29 @@ let getAuthInfo = (~email_token=None, json) => {
   }
   totpInfo
 }
+
+let getEmailTmpToken = () => {
```

Comment:
what are trying to do over here

Reply:
The logic for this function is that if the value is passed in the function params it will take that value or it will take the value present in the local storage . Changed the logic of this function

Reply:
rather than depending on loalstorage we should take the value from Prelogin ENUM

Reply:
But wont it affect the cases where the page is refreshed and the context also is set to default and next user has to immediately set to PreLogin state

Reply:
Changed Jeeva